test/ui/starter-select.test.ts

```diff
@@ -46,7 +46,7 @@ describe("UI - Starter select", () => {
       currentPhase.end();
     });
     game.onNextPrompt("SelectStarterPhase", UiMode.STARTER_SELECT, () => {
-      const handler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+      const handler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const handler = game.scene.ui.getCurrentHandler<StarterSelectUiHandler>();
```